src/views/shared/amount-input/amount-input.view.jsx

```diff
@@ -162,7 +162,7 @@ function AmountInput(Component) {
       const maxAmountWithoutFeeInFiat = convertAmountToFiat(maxAmountWithoutFee);
 
       if (showInFiat) {
-        setValue(maxAmountWithoutFeeInFiat);
+        setValue(maxAmountWithoutFeeInFiat.toString());
```

Comment:
Handling amounts as strings is weird, each calculation you need to do you'll need to parse string to a number.  Receiving string as input makes more sense from the UI point of view, but not for handling the data inside the component. That's why the component already has a `value` property, which is the UI representation of the `amount.fiat` with little variations (like it could be just `''`, an amount cannot).

Reply:
Ah! You are right, I thought that I read it inside the `setAmount`. Then it's perfect :)